src/AppBundle/Controller/TicketController.php

```diff
@@ -109,9 +109,10 @@ public function addMessageAction(Request $request, int $id)
         $loggedUser = $this->getLoggedUser();
 
         $data = TicketDto::fromArray($request->request->all());
-        $sender = $this->get('domain.adminNotifications.sender');
+        $adminSender = $this->get('domain.adminNotifications.sender');
+        $userSender = $this->get('domain.userNotifications.sender');
```

Comment:
Sarebbe meglio uniformare i servizi in  modo da non avere duplicazione di codice